redhat-cop/tower_utilities
# Add support for OCP based install

### What does this PR do?

This PR adds a role for installing AAP on OpenShift.

This is a fairly basic role at this point. I plan on adding support for installing/configuring RH-SSO integration with AH in future PR(s). 

I'd also like feedback/thoughts on whether the role should support creating multiple AC and/or AH instances (in one or more namespaces) before the PR is merged because the variable structure will need to change slightly (which would be a breaking change I'd like to avoid if possible).

### How should this be tested?

Create a playbook to run the `aap_ocp_install` role with appropriate variables set (see the README.md in the role).

### Is there a relevant Issue open for this?

No

### Other Relevant info, PRs, etc

None
